src/DateTimePicker.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+import Flatpickr from 'flatpickr';
+
+/**
+ * @module DateTimePicker
+ */
+class DateTimePicker extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			value: props.value || '',
+			isSupported: false
```

Comment:
whether or not datetime-local is supported natively. I'll make this round of changes and also try to clarify some things.